### PR TITLE
Change default blocking and NON-REF LOD params for M2 GVCF mode

### PR DESCRIPTION
**Pull request by ldgauthier**
Teeny tiny PR -- not happy with the AF thresholding for GVCF mode yet

### CODE REVIEW

**Comment by ldgauthier** [comment]
@droazen I'd like this to squeeze into 4.1.  Tests pass locally after the second commit.

**Comment by droazen** [comment]
@davidbenjamin please review

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java [hunk 1]
```diff
@@ -251,12 +251,12 @@ public double getInitialLod() {
             "(must be specified in increasing order)", optional = true)
     public List<Double> GVCFGQBands = new ArrayList<>(70);
     {
-        for (double i = -4.0; i <= 1; i = i + 0.5) {
+        for (double i = -2.5; i <= 1; i = i + 0.5) {
```
Comment (davidbenjamin) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java]: Could you use `GVCFGQBands = Arrays.asList(MathUtils.createEvenlySpacedPoints(-2.5, 1, 8))`?

**Comment by ldgauthier** [comment]
I'm going to keep the initialization as is because it mirrors HaplotypeCaller and because I feel like defining blocks by their size is more intuitive than defining the number of blocks between two bounds.

**Pull request merged**